sources/p987.6/document.adoc

```diff
@@ -0,0 +1,36 @@
+= Draft Recommended Practice for Preparing an IEEE Standards Draft
+:docnumber: P987.6/D3
+:doctype: standard
+:copyright-year: 2020
+:revdate: 2021-01-01
+:confirmed-date: 2021-01-01
+:language: en
+:society: Template Society
+:committee: Standards Staff Engineering Committee
+:working-group: Working Group Name
+:wg_chair: Arthur C. Clark
+:wg_vicechair: Alessandro Volta
```

Comment:
All document attributes use `-` hyphens not `_` underscores. This was an issue that was reverted in metanorma-ieee.

Reply:
Fixed.